compiler/rustc_codegen_cranelift/src/vtable.rs

```diff
@@ -21,7 +18,7 @@ pub(crate) fn drop_fn_of_obj(fx: &mut FunctionCx<'_, '_, '_>, vtable: Value) ->
         pointer_ty(fx.tcx),
         vtable_memflags(),
         vtable,
-        (DROP_FN_INDEX * usize_size) as i32,
+        (ty::COMMON_VTABLE_ENTRIES_FIRST_OFFSET_METADATA_DROPINPLACE * usize_size) as i32,
```

Comment:
That's a very long name :)

Reply:
I'm not good at naming things in English. Suggestions welcome!

Reply:
Changed according to bjorn3's suggestion